style/components/badge.css

```diff
@@ -0,0 +1,99 @@
+.badge-div {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  width: 12rem;
+  border-bottom: 1px solid rgb(156, 156, 156);
+  padding: 10px;
+  margin-bottom: 10px;
+}
+
+.txt-badge {
+  background-color: #0d8dc9;
+  padding-left: 0.6rem;
+  padding-right: 0.6rem;
+  padding-top: 0.3rem;
+  padding-bottom: 0.3rem;
+  border-radius: 50%;
+  position: absolute;
+  top: -8%;
+  right: -2%;
+  color: white;
+  font-size: 12px;
+  font-weight: 550;
+}
+
+.btn-badge {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  /* position: absolute; */
```

Comment:
You are using similar CSS properties in different classes instead of that you can create a utility class for the repeated CSS.

Reply:
Following the OOCSS approach, will make your CSS shorter and easily maintainable.

Reply:
sure I will try this